api/v3/CustomField.php

```diff
@@ -235,7 +235,7 @@ function _civicrm_api3_custom_field_validate_field($fieldName, $value, $fieldDet
       break;
 
     case 'Float':
-      if (!CRM_Utils_Rule::numeric($value)) {
+      if (!is_numeric($value)) {
```

Comment:
The function this is in has been deprecated for 11 years. I'm amazed you let it live. https://github.com/civicrm/civicrm-core/commit/c490a46ac4bf490d3fbb31e35d24c6b43e764f00#diff-f83c880121c0bc94937da675e5076762bbc03b2d2379decdb07cc5d0966ad1faL157-R148

(That link takes about 2 minutes to load since it's a giant commit.)